packages/manager/apps/web/client/app/hosting/database/order-private/hosting-database-order-private.html

```diff
@@ -0,0 +1,155 @@
+<oui-back-button
+    data-on-click="$ctrl.goBack()">
+    <span data-translate="hosting_database_order_private_title"></span>
+</oui-back-button>
+
+<div data-ovh-alert="{{alerts.order}}"></div>
+
+<div class="text-center"
+        data-ng-if="$ctrl.loading.init">
+    <oui-spinner data-size="l"></oui-spinner>
+</div>
+
+<div data-ng-if="!$ctrl.loading.init">
+    <p data-translate="hosting_database_order_private_description"></p>
+
+    <oui-stepper name="sqlDatabaseOrderForm"
+        data-current-index="$ctrl.currentIndex"
+        data-on-finish="$ctrl.checkout()">
+
+        <!-- DB Type -->
+        <oui-step-form
+            data-header="{{:: 'hosting_database_order_private_step_header_type' | translate }}"
+            data-editable="$ctrl.isEditable && !$ctrl.checkoutLoading">
+            <oui-field
+                data-label="{{:: 'hosting_database_order_private_type' | translate }}">
+                <oui-radio
+                    name="orderType"
+                    data-description="{{:: 'hosting_database_order_private_type_text' | translate }}"
+                    data-model="$ctrl.order.type"
+                    data-value="'privateSQL'"
```

Comment:
Even if it's static, this should be in a constant probably, especially as the value is also used in the controller